frameworks/kafka/docs/configure.md

```diff
@@ -27,82 +27,53 @@ The Kafka scheduler runs as a Marathon process and can be reconfigured by changi
 1.  Go to the `Services` tab of the DC/OS web interface.
 1.  Click the name of the Kafka service to be updated.
 1.  Within the Kafka instance details view, click the menu in the upper right, then choose **Edit**.
-1.  In the dialog that appears, click the **Environment** tab and update any field(s) to their desired value(s). For example, to [increase the number of Brokers][8], edit the value for `BROKER_COUNT`. Do not edit the value for `FRAMEWORK_NAME` or `BROKER_DISK` or `PLACEMENT_STRATEGY`.
-1.  A `PHASE_STRATEGY` of `STAGE` should also be set. See "Configuration Deployment Strategy" below for more details.
+1.  In the dialog that appears, click the **Environment** tab and update any field(s) to their desired value(s). For example, to [increase the number of Brokers][8], edit the value for `BROKER_COUNT`. Do not edit the value for `FRAMEWORK_NAME` or `BROKER_DISK`.
+1.  Choose a `DEPLOY_STRATEGY`: serial, serial-canary, parallel-canary, or parallel. See the SDK Developer guide for more information on [deployment plan strategies](https://mesosphere.github.io/dcos-commons/developer-guide.html#plans). <!-- I'm not sure I like this solution, since users aren't going to have the context for the dev guide). -->
```

Comment:
Previous version has phase_strategy INSTALL and STAGE.  STAGE is equal to serial-canary. INSTAL is equal to serial. 

We may again keep the original explanation, saying that serial installation is our default. If you want decision points, use canary strategies. 

serial-canary: after two decision points, it behaves like serial
parallel-canary: after two decision points, it behaves like parallel (installs all parallely)